monai/utils/stopperutils.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+def stopping_fn_from_metric(metric_name):
```

Comment:
It's not very clear to return another function in a function. Suggest to use class object with callback function:
```py
class stop_from_metric:
    __init__(self, metric_name):
        self.metric_name = metric_name
    def __callable__(self):
        return engine.state.metrics[self.metric_name]
```

Thanks.

Reply:
@Nic-Ma . Thanks for the comment. 

Returning functions is very common in python. This is how decorators are implemented.

Also ignite specifically asks for a function in documentation so passing a callable class can be confusing IMO. What do you think?

Reply:
Hi @madil90 ,

OK, ignite requires a "score_function: Callable", I am OK to keep your current function in function.
Maybe we need to refactor all utils later.
Thanks.